torchgeo/datamodules/vaihingen.py

```diff
@@ -23,26 +27,44 @@ class Vaihingen2DDataModule(pl.LightningDataModule):
 
     def __init__(
         self,
-        batch_size: int = 64,
+        train_batch_size: int = 32,
         num_workers: int = 0,
+        patch_size: Tuple[int, int] = (64, 64),
+        num_patches_per_tile: int = 32,
         val_split_pct: float = 0.2,
         **kwargs: Any,
     ) -> None:
         """Initialize a LightningDataModule for Vaihingen2D based DataLoaders.
 
         Args:
-            batch_size: The batch size to use in all created DataLoaders
+            train_batch_size: The batch size used in the train DataLoader
+                (val_batch_size == test_batch_size == 1). The effective batch size
+                will be 'train_batch_size' * 'num_patches_per_tile'
```

Comment:
Can you also clarify this in `OSCDDataModule`?

Reply:
Caleb suggested in #876 a different approach and naming of this batch_size parameter. Should I change this PR accordingly?

Reply:
Added a comment there, we should decide on a final naming scheme in #876 and then use the same scheme here.